app/controllers/AccountHomeController.scala

```diff
@@ -75,9 +80,18 @@ class AccountHomeController @Inject() (
   def startApplication: Action[AnyContent] =
     identify.async {
       implicit request =>
-        for {
-          draftId <- counterRepository.nextId(CounterId.DraftId)
-          _       <- userAnswersService.set(UserAnswers(request.userId, DraftId(draftId)))
-        } yield Redirect(navigator.startApplicationRouting(request.affinityGroup, DraftId(draftId)))
+        AuthUserType(request) match {
+          case None           =>
+            // TODO: Update error handling
+            Future.successful(Redirect(controllers.routes.JourneyRecoveryController.onPageLoad()))
+          case Some(authType) =>
+            for {
+              nextId      <- counterRepository.nextId(CounterId.DraftId)
+              draftId      = DraftId(nextId)
+              userAnswers  = UserAnswers(request.userId, draftId, lastUpdated = Instant.now(clock))
+              userAnswers <- userAnswers.setFuture(ApplicantUserType, authType)
+              _           <- userAnswersService.set(userAnswers)
+            } yield Redirect(navigator.startApplicationRouting(request.affinityGroup, draftId))
```

Comment:
I think we can refactor the navigator now that we have this change. We don't need this special `startApplicaitonRouting` method, instead we can remove it and rely on the `navigate` method. What do you think?

Reply:
I'm partial to this routing method - while it would match the rest of the file, I do think the additional work to use the `nextPage` method is worth it (i think that's what you're referring to?)